code/game/objects/structures.dm

```diff
@@ -87,7 +87,14 @@
 	structureclimber = user
 	if(do_mob(user, user, adjusted_climb_time))
 		if(src.loc) //Checking if structure has been destroyed
-			if(do_climb(user))
```

Comment:
The original climb should be the first if check, since the majority of players won't be simians.

Reply:
Not really sure what you mean here, putting this after would make them climb